javascript/attributes.js

```diff
@@ -59,11 +65,6 @@ export const extractElementAttributes = element => {
     attrs.value = collectedOptions.join(',')
   } else {
     attrs.value = element.value
-    if (element.tagName.match(/select/i)) {
```

Comment:
@julianrubisch I'm >99% sure that this code was intended to be deleted when you added support for `values` but it forgot to die.